src/sentry/templatetags/sentry_api.py

```diff
@@ -21,14 +20,12 @@ def serialize(context, obj):
     else:
         user = None
 
-    return convert_to_json(serialize_func(obj, user), escape=True)
+    return convert_to_json(serialize_func(obj, user))
 
 
 @register.simple_tag
-def convert_to_json(obj, escape=False):
-    if escape:
-        return json.dumps_htmlsafe(obj)
-    return mark_safe(json.dumps(obj))
+def convert_to_json(obj):
```

Comment:
Do we not have cases where we don't want to escape JSON in templates? My hunch is "no", but I don't have anything to back that up.